pages/clubs-guides/clubs-guides.jsx

```diff
@@ -0,0 +1,99 @@
+var React = require('react');
+var HeroUnit = require('../../components/hero-unit.jsx');
+
+var GuideList = React.createClass({
+  getInitialState: function(){
+    return {
+      guides : []
+    }
+  },
+  setData : function(data){
+    this.setState({guides : data });
+  },
+  fetchJSON : function(path,callback){
+    var httpRequest = new XMLHttpRequest();
+    httpRequest.onreadystatechange = function() {
+      if(httpRequest.readyState === 4) {
+        if(httpRequest.status === 200) {
+          var data = JSON.parse(httpRequest.responseText);
+          if (callback) callback(data);
+        }
+      }
+    };
+    httpRequest.open('GET', path);
+    httpRequest.send();
+  },
+  componentDidMount: function() {
+    this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
+  },
+  render: function(){
+    var categories = this.state.guides.map(function(e){
+
+      var guideLinks = e.guides.map(function(f){
+        var translationLinks = [];
+        if(f.translations && f.translations.length > 0){
+          translationLinks = f.translations.map(function(g){
+            return (
+              <li>
+                <a href={ g.url }>{ g.title }</a><span className="language"> - ({ g.language })</span>
+              </li>
+            );
+          });
+        }
+
+        var isHidden = true;
+
+        if(translationLinks.length > 0){
+          var translationUI = translationLinks;
```

Comment:
I'm also not sure why it needs to be assigned to `translationUI`. Can't you just use `translationLinks` directly?